pack.yaml

```diff
@@ -1,10 +1,9 @@
 ---
```

Comment:
According to the docs it's not required. It will take preference over `name` when given. i noticed some strange behaviour when using both with different values, but had no real issues.